### ISSUE TITLE
package_files command does not have an option to preserve symlinks

### ISSUE THREAD

**Issue by kdsx** [conan-io/conan#1312]
This option would be very useful for packing shared libraries which often have a couple of symlinks:
```
libssl.so.1.0.0
libssl.so.1 -> libssl.so.1.0.0
libssl.so -> libssl.so.1.0.0
```

**Comment by memsharded** [comment]
Sure, this was missing in the initial implementation. Lets try to add it for next 0.24 release. Thanks for the feedback!

**Comment by lasote** [comment]
Merged to develop, will be available in the next 0.25

**Comment by memsharded** [comment]
Released in 0.25
